static/styles/popovers.css

```diff
@@ -36,6 +36,11 @@
     text-align: center;
 }
 
+.popover_info li {
+    text-overflow: ellipsis;
+    overflow: hidden;
+}
```

Comment:
I fixed a couple styling issues with whitespace and hover behavior; this is now:
```
+.popover_info .custom_user_field {
+    text-overflow: ellipsis;
+    overflow: hidden;
+
+    a i {
+        /* Override margin on top-level icons */
+        margin-right: 0;
+    }
+
+    a:hover {
+        text-decoration: none;
+    }
+}
```